governance/sponsors_partners.md

```diff
@@ -0,0 +1,110 @@
+(gov_sponsors_partners)=
+# Sponsors and institutional partners
+The Random Variables Council are the primary leadership for the project.
+No outside institution, individual or legal entity has the ability to
+own, control, usurp or influence the project other than by participating in the Project
+as Contributors and Council Members.
+However, because financial and institutional support is an important funding mechanism for the project,
+it is important to formally acknowledge it.
+
+ArviZ uses two figures to acknowledge financial and institutional support:
+sponsors and institutional partners. Both can have "active" and "inactive" status.
+
+We use the sponsor figure for entities who provide direct financial support,
+institutional partners for entities who employ an ArviZ contributor and
+let them work on the project as part of their official duties.
+
+(gov_sponsors)=
+## Sponsors
+Sponsors are entities that provide direct financial support.
+The minimum level of support given to ArviZ to be considered a Sponsor are:
+
+* $10,000/yr for unrestricted financial contributions (e.g., donations)
+* $20,000/yr for financial contributions for a particular purpose (e.g., grants)
```

Comment:
We should add CZI as they are meeting this level

Reply:
I have added them already,  and will also add  all institutional partners once there is consensus on this way to awknowledge them.

Small logo in homepage: https://arviz-governance--3.org.readthedocs.build/en/3/
Bigger logo and funding details in sponsors and partners page: https://arviz-governance--3.org.readthedocs.build/en/3/sponsors_partners.html